docker/dev/backend/Dockerfile

```diff
@@ -37,6 +37,9 @@ RUN echo "deb [signed-by=/etc/apt/keyrings/nodesource.gpg] https://deb.nodesourc
 RUN apt-get update
 RUN apt-get install -y nodejs
 
+# Setup java. Needed for https://github.com/NUARIG/ladle which is used in specs to test LDAP related behavior.
+RUN apt-get install -y openjdk-17-jdk
```

Comment:
:question: For me to understand: That's not a new dependency you introduced in this PR, but rather one that you discovered while running tests that no one else has run before in docker?